docker-compose.yml

```
@@ -0,0 +1,12 @@
+version: '2'
+services:
+  db:
+    image: mysql:latest
+    restart: always
+    ports:
+      - 3306:3306
+    environment:
+      MYSQL_ROOT_PASSWORD: root
```

Nie wprowadzajmy od razu złych nawyków: NIGDY nie commitujemy haseł. Testowych, jak testowych, ale zapomnisz się i zacommitujesz hasło produkcyjne. NIGDY to NIGDY nie zapomnisz. Kategoryczny zakaz commitowania jakichkolwiek danych wrażliwych: nazwy użytkowników, haseł etc.
użyj:
```suggestion
      MYSQL_ROOT_PASSWORD: ${WEDDING_HALL_DB_ADMIN_PASSWORD}
```